tensorflow/python/ops/math_ops.py

```diff
@@ -4229,6 +4229,25 @@ def polyval(coeffs, x, name=None):
      p(x) = coeffs[n-1] + x * (coeffs[n-2] + ... + x * (coeffs[1] +
             x * coeffs[0]))
 
+  Usage Example:
+
+  >>> y = tf.math.polyval([2, 1, 0], 3) # evaluates 2 * (3**2) + 1 * (3**1) + 0 * (3**0)
+  >>> print(y)
+  tf.Tensor(21, shape=(), dtype=int32)
+
+  `tf.math.polyval` can also be used in polynomial regression. Taking
+  advantage of this function can facilitate writing a polynomial equation
+  as compared to explicitly writing it out, especially for higher degree
+  polynomials.
+
+  >>> x = tf.constant(3)
+  >>> theta1 = tf.Variable(2)
+  >>> theta2 = tf.Variable(1)
+  >>> theta3 = tf.Variable(0)
+  >>> y = tf.math.polyval([theta1, theta2, theta3], x)
+  >>> print(y)
+  tf.Tensor(21, shape=(), dtype=int32)
```

Comment:
You don't need to assign to `y` and `print(y)`. How about

```python
  >>> tf.math.polyval([theta1, theta2, theta3], x)
￼  tf.Tensor(21, shape=(), dtype=int32)
```

Reply:
Sure.

Reply:
Well, I meant to do this throughout the PR, not only on this instance. There is at least one more place you have to change